packages/core/admin/ee/server/register.js

```diff
@@ -4,8 +4,31 @@ const { features } = require('@strapi/strapi/lib/utils/ee');
 const executeCERegister = require('../../server/register');
 const createAuditLogsService = require('./services/audit-logs');
 
+// Migrate the audit logs table name for users coming from v4.6.0
+const migrateAuditLogsTable = async ({ oldContentTypes, contentTypes }) => {
+  // Check if the table name needs to be migrated
+  const oldName = oldContentTypes?.['admin::audit-log']?.collectionName;
+  const newName = contentTypes['admin::audit-log']?.collectionName;
+  const shouldMigrate = oldName === 'audit_logs' && newName === 'strapi_audit_logs';
+
+  if (shouldMigrate) {
+    // Migrate the name audit logs table
+    if (await strapi.db.getSchemaConnection().hasTable('audit_logs')) {
+      await strapi.db.getSchemaConnection().renameTable('audit_logs', 'strapi_audit_logs');
```

Comment:
I wanted to store the schema connection in a variable, but it would throw errors, it looks like I have to get a fresh one every time